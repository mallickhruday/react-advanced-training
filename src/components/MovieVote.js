import React from 'react'
import styled from 'styled-components'

const Count = styled('span')`
  font-size: 0.8em;
`

const MovieVote = ({ movie }) => (
  <>
    {movie.vote_average} / 10 <Count>({movie.vote_count} votes)</Count>
  </>
)
export default MovieVote
