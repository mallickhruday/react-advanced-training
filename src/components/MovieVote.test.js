import React from 'react'
import renderer from 'react-test-renderer'
import MovieVote from './MovieVote'

describe('MovieVote', () => {
  it('should render vote', () => {
    const component = renderer.create(
      <MovieVote movie={{ vote_average: 5.4, vote_count: 2123 }} />,
    )
    expect(component.toJSON()).toMatchSnapshot()
  })
})
