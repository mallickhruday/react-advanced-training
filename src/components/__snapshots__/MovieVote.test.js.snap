// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieVote should render vote 1`] = `
Array [
  "5.4",
  " / 10 ",
  .c0 {
  font-size: 0.8em;
}

<span
    className="c0"
  >
    (
    2123
     votes)
  </span>,
]
`;
